### ISSUE TITLE
Are the hyper-parmaters same for different datasets?

### ISSUE THREAD

**Issue by zrfreya** [rosinality/swapping-autoencoder-pytorch#10]
What's the learning rate,  gradient penalty weight, training epochs for the Church, flickr waterfall/mountains, and other datasets?

**Comment by rosinality** [comment]
It seems like that authors have used same hyperparameters except image resolutions and batch sizes.

**Comment by zrfreya** [comment]
> It seems like that authors have used same hyperparameters except image resolutions and batch sizes.

Could you please share the pretrained weights of the Church dataset? I have trained the network with the Church dataset. However, my results are worse than the results in your paper.

![Uploading 009bc93861ffc9d11c906f8785cb95409fe92f79_to_00b02653aba8d23a0d4d8cd065bb1cad7e3ea0ba.png…]()
![002672fff730e9216f9bd07d7e9be58849e04ef5_to_0052e05e97a799402b5da686943d7ca42d21ff22](https://user-images.githubusercontent.com/17635831/92698340-d6006380-f37e-11ea-809b-7deb7262d49d.png)

**Comment by rosinality** [comment]
Actually I am not the author of the paper.

**Comment by zrfreya** [comment]
> Actually I am not the author of the paper. Sorry for confusion.

Oops. Sorry for my mistake. Thanks for sharing your great work.
